src/easydmp/lib/upgrade/management/commands/resetmigrationhistory.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.db import DEFAULT_DB_ALIAS, connections
+from django.db.migrations.recorder import MigrationRecorder
+
+
+class Command(BaseCommand):
+    help = "Reset migration history by emptying the django_migrations table"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--database',
+            default=DEFAULT_DB_ALIAS,
+            help='Select which database to use. Defaults to the "default" database.',
+        )
+
+    def handle(self, *args, **options):
+        # Get the database we're operating from
+        db = options['database']
+        connection = connections[db]
+
+        # Hook for backends needing any database preparation
+        connection.prepare_database()
+
+        # Get the django_migrations table wrapper
+        recorder = MigrationRecorder(connection)
+
+        # Store existing state
+        ordered_state = recorder.migration_qs.values_list('app', 'name').order_by('applied')
```

Comment:
@hmpf Does this statement do anything?

Reply:
It's dead code. At first I had planned to also do the fake-migrations, and then it would have been useful to check state before and after.